src/richtext/utils/hastToRichText.ts

```diff
@@ -127,9 +63,9 @@ const VFILE_RULE = "failed-to-serialize-node";
 const VFILE_SOURCE = "prismic";
 
 /**
- * Configuration that determines the output of `hastUtilToRichText`.
+ * Configuration that determines the output of `toRichText`.
  */
-export type HastUtilToRichTextConfig = {
+export type HASTToRichTextConfig = {
```

Comment:
💡 #idea: I think this type should be written as `HastToRightTextConfig` since hast is not an acronym like HTML.

Reply:
From GitHub, it's the acronym for Hypertext Abstract Syntax Tree, so I'll leave it as-is, but we can rename it later.

See repository description: https://github.com/syntax-tree/hast